.gitlab-ci.yml

```diff
@@ -1,9 +1,8 @@
 build:
-  image: fedora:26
+  image: registry.centos.org/minishift/minishift-build:latest
```

Comment:
Needs https://github.com/CentOS/container-index/pull/331 to be merged

Reply:
@gbraad we should call it `minishift-builder` image , refer https://github.com/CentOS/container-index/pull/331#discussion_r186370023